src/drivers/driver_zephyr.c

```diff
@@ -50,7 +50,9 @@ void wpa_supplicant_event_wrapper(void *ctx,
 				char *ies = os_zalloc(data->auth.ies_len);
 
 				if (!ies) {
-					wpa_printf(MSG_ERROR, "%s: Failed to alloc ies\n", __func__);
+					wpa_printf(MSG_ERROR,
+					  "%s:%d Failed to alloc ies: (%d)bytes\n",
```

Comment:
Small nit. The output string looks a bit hard to read, why not have `...: %d bytes\n` instead?